### DISCUSSION TITLE
Axmol 2.1.3 create button with image wrong color

### DISCUSSION THREAD

**Discussion by NgVThangBz** [axmolengine/axmol#1947] [Q&A] [answered]
i create button with Image:
button displayed in green color but Image is black.

Does anyone have the same problem, I used other images but it didn't have any problem, however with the 2 images below it got error, you can see the image and my code below :
(I didn't encounter it in the previous version nor the beta)
![Screenshot 2024-05-31 at 08 47 52](https://github.com/axmolengine/axmol/assets/125545428/3b55e875-3d95-4f55-8da0-74ee15264e8e)
![image](https://github.com/axmolengine/axmol/assets/125545428/2b1f34b0-cce3-4b6c-87f7-bce63c16220a)

![close](https://github.com/axmolengine/axmol/assets/125545428/2231244a-5184-407c-b3bb-1c00439a23c3)
![circle](https://github.com/axmolengine/axmol/assets/125545428/750f6699-57f7-4033-b112-5bae2b4ad2fc)

**Comment by rh101** [comment]
Which platform is this running on? Windows? iOS? Android? etc..

**Reply by NgVThangBz** [reply]
i use IOS and Android (Haven't tested on Windows yet)

**Reply by rh101** [reply, marked as answer]
The images being used are in an unsupported pixel format.  Change the files to RGBA (32bit), and they will work.

**Comment by NgVThangBz** [comment]
whether axmol can convert formats to RGBA, for example R8,
Currently I am loading the image in R8 format, and it has an alpha error
Is there any solution other than changing the image?
